pandas/io/tests/test_parsers.py

```diff
@@ -3070,6 +3070,26 @@ def test_whitespace_lines(self):
         df = self.read_csv(StringIO(data))
         tm.assert_almost_equal(df.values, expected)
 
```

Comment:
I would put both of these in a new ``TestNetwork`` (below TestS3), or just throw them in there ok too. You can have some of the boilerplate in the ``setUp`` creation call. Eg. the filepath to salary.